src/tests/integration/test_duckDB.py

```diff
@@ -5,7 +5,56 @@
 import pandas as pd
 
 
-from sql.warnings import JupySQLDataFramePerformanceWarning
+@pytest.mark.parametrize(
+    "method, expected_type, expected_native_method",
+    [
+        ("DataFrame", pd.DataFrame, "df"),
+        ("PolarsDataFrame", pl.DataFrame, "pl"),
+    ],
+)
+@pytest.mark.parametrize(
+    "autocommit",
+    [True, False],
+)
+def test_sqlalchemy_connection_converts_to_data_frames_natively(
+    monkeypatch,
+    ip_with_duckdb_sqlalchemy_empty,
+    method,
+    expected_type,
+    expected_native_method,
+    autocommit,
+):
+    ip_with_duckdb_sqlalchemy_empty.run_cell(
+        f"%config SqlMagic.autocommit = {autocommit}"
+    )
+
+    ip_with_duckdb_sqlalchemy_empty.run_cell(
+        "%sql CREATE TABLE weather (city VARCHAR, temp_lo INT);"
+    )
+    ip_with_duckdb_sqlalchemy_empty.run_cell(
+        "%sql INSERT INTO weather VALUES ('San Francisco', 46);"
+    )
+    ip_with_duckdb_sqlalchemy_empty.run_cell(
```

Comment:
<img width="664" alt="Screenshot 2023-07-31 at 1 45 14 PM" src="https://github.com/ploomber/jupysql/assets/29096135/3c6b8753-44ca-4bc9-9b84-4ea007b31e00">

Why is this printing `Count = 1` ?

Reply:
duckdb returns the count of inserted values and we only return the output of the latest command, so one row inserted